docs/guides/building/linux.md

```diff
@@ -63,6 +63,11 @@ If your app plays audio/video you need to enable `tauri.conf.json > tauri > bund
 
 :::
 
+## Snap Package
+
+Snap packages are an alternative distribution format that also does not rely on the system installed packages. Snap was created by Canonical. Snap packages work across most linux distributions. In order to build snap packages using the CLI, the `snapcraft` package must be installed. On the other hand, users are required to have the `snapd` package installed on their system. After building your snap package, it can be signed and uploaded to the official Snap Store for easier distribution. The output file can be installed using (`sudo snap install ./MyProject.snap --dangerous`). The dangerous flag is required to install the package. In order to bypass the --dangerous flag and sign your package, a Snap Store account is required and you must login via. snapcraft on your local system using (`snapcraft login`).
```

Comment:
I feel this could be reworded a bit. For example, "distribution format that *also* does not rely" presumably refers to the previous AppImage section, and each section should be able to be read without having read the one before it. Also, the sentences "the `snapcraft` package must be installed. On the other hand, users are required to have the `snapd` package installed on their system" is pretty confusing. What is it "on the other hand" of?